app/gateway/changelog.md

```diff
@@ -11,6 +11,55 @@ For Kong Gateway OSS, view the [OSS changelog on GitHub](https://github.com/Kong
 
 For product versions that have reached the end of sunset support, see the [changelog archives](https://legacy-gateway--kongdocs.netlify.app/enterprise/changelog/).
 
+## 3.7.1.0
+**Release Date** 06/17/2024
+
+### Features
+#### Plugins
+
+* [**Request Validator**](/hub/kong-inc/request-validator/) (`request-validator`)
+  * Added the new configuration field `content_type_parameter_validation` to determine whether to enable Content-Type parameter validation.
+
+### Fixes
+#### Core
+
+* **DNS Client**: Fixed an issue where the Kong DNS client stored records with non-matching domain and type when parsing answers.
+It now ignores records when the RR type differs from that of the query when parsing answers.
+* Fixed an issue where the `host_header` attribute of the upstream entity wouldn't be set correctly as a Host header in requests to the upstream during connection retries.
+* Built-in RBAC roles for admins (`admin` under the default workspace and `workspace-admin` under non-default workspaces) now disallow CRUD actions to `/groups` and `/groups/*` endpoints.
+* Fixed an issue where the priority field could be set in a traditional mode route when `router_flavor` was configured as `expressions`. 
+
+#### Plugins
+
+* [**AI Proxy**](/hub/kong-inc/ai-proxy/) (`ai-proxy`)
+  * Resolved an issue where the object constructor would set data on the class instead of the instance.
+
+<!-- 
+* [**Basic Authentication**](/hub/kong-inc/basic-auth/) (`basic-auth`)
+  * Fixed an issue where the realm field wasn't recognized for older kong versions (before 3.6)
+
+* [**Key Authentication**](/hub/kong-inc/key-auth/) (`key-auth`)
+  * Fixed an issue where the realm field wasn't recognized for older kong versions (before 3.7)
+ -->
```

Comment:
These entries are commented out because they don't seem to apply to this version. Waiting to hear some feedback on that.